Fix Multiple Issues in Media Manager

suxur opened the pull request:
Issues Fixed:
- When creating folders in Firefox the name is "undefined"
- When moving files in Firefox the selected destination returns "undefined"
- Unable to use "command" or "meta" key for multi-selection of files
- Using "meta key" brought to surface another issue where you could select the same file / folder multiple times.

Commit:
Add missing semi-colons

Commit:
Allow cherry picking multi-select in media manager

Prevent same file/folder from being selected multiple times

Commit:
Fix issues when creating folders and moving files in Firefox

When trying to create a folder or moving a file, the name or destination
would return "undefined" when using Firefox. This replaces getting the
input value via jQuery and instead using Vue and it's v-model binding.

emptynick commented on resources/views/media/manager.blade.php:
```diff
@@ -293,7 +293,8 @@
                     <h4>{{ __('voyager::media.destination_folder') }}</h4>
-                    <select class="form-control">
+                    <select class="form-control" v-model="modals.move_files.destination">
+                        <option value="">Select Destination</option>
```
Any specific reason for this option?
What happens if I use it?

suxur replied:
Mainly for placeholder text. When using v-model on the select input we have a default value, currently set to `''` and if this option is not there the select input appears empty, this prevents that and gives the user instruction.

Currently, if used it tries to move the file or folder to the same location it already exists which returns an error "File already exists at path." If you're okay with keeping this as a "placeholder" option, I'd like to update this PR by adding a check the value before submitting the post request to move the file.

Commit:
Fix scope reference to `this`

Commit:
Prevent empty field being submitted when moving files

Add "disabled" to move file default option. This prevents someone from
selecting this option after they have selected another option. Check
for an empty destination before submitting the post request to move
the file or folder.

emptynick commented on resources/views/media/manager.blade.php:
```diff
@@ -293,7 +293,8 @@
                     <h4>{{ __('voyager::media.destination_folder') }}</h4>
-                    <select class="form-control">
+                    <select class="form-control" v-model="modals.move_files.destination">
+                        <option value="" disabled>Select Destination</option>
```
This hardcoded string needs to be translated.
For simplicity I suggest 
```suggestion
                        <option value="" disabled>{{ __('voyager::media.destination_folder') }}</option>
```

emptynick committed:
Replace hardcoded string with translation function

Co-Authored-By: suxur <suxur@me.com>

Commit:
Fix reference to image modal

Double clicking an image did not open the image modal

Commit:
Fix selected media count when using shift select

emptynick committed:
Merge branch '1.2' into 1.2
